### DISCUSSION TITLE
How to control when it updates?

### DISCUSSION THREAD

**Discussion by thany** [qdm12/ddns-updater#876] [Q&A]
I'm not sure how it currently decides whether or not to update, but I just want it to send a daily update. Nothing more.

I mean, checking the public IP address requires a call anyway, periodically (I still don't know, or control, how often this happens), so it might as well be an update. Whether doing a HTTP call to check the IP-address, or do a HTTP call to *update* the IP-adress, it's a HTTP call in both cases. So you might as well skip the former and *only* ever to the latter, daily.

So how do I go about configuring that?

**Comment by harryfine** [comment]
Same question, should you just run it through cron?
